docs/vsphere_ci/README.md

```diff
@@ -0,0 +1,178 @@
+# Building Windows VM images in vSphere
+
+## Goals
+This document focuses on building Windows images in vSphere 6.7 environment used in CI but can be used as example for
+other vSphere environments. We use Packer to automate installation and configuration of the Windows images. Packer 
+generates a vSphere VM template that can be converted to a VM. After the VM has been created, the machine-api can clone
+it multiple times as part of the Windows VM creation. The following steps need to be executed from the bastion instance
```

Comment:
Please remove `multiple times`. It is cloned only once.

Reply:
I meant machine-api can clone a Windows VM multiple times for multiple Windows VMs. I can remove it if it is confusing.